bin/upload_buildkite_test_analytics_junit

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+
+FILE_PATH=$1
+ANALYTICS_TOKEN=$2
+
+if [ -z "$FILE_PATH" ]; then
+	echo "You must pass a path to the file you want to upload to Test Analytics"
+	exit 1
+fi
+
+if [ -z "$ANALYTICS_TOKEN" ]; then
+	echo "You must pass a Buildkite Analytics Token"
+	exit 1
+fi
+
+if [ ! -f "$FILE_PATH" ]; then
+	echo "No file exists at $FILE_PATH. Cancelling Upload"
+	exit 0
```

Comment:
What is the reason you picked `0` for the exit status? I assume it was a conscious choice, so I think it'd be good to document that to avoid confusion - like the one I am having now 😅

Reply:
Hmm so there are two questions here I guess – the reason we're exiting cleanly was a decision that I went back-and-forth on, but eventually figured that it might be nicer to not have to have file existence logic in the parent script – we could just skip it if it's not found. 

The reason I used `exit 0` to indicate "exit cleanly" was just habit – sure it's identical to `exit` but when reading it's IMHO extra-clear that there's no exit error code.

Is this a good way to do it? Maybe not?

I could document it, but like...it's a style thing not functional thus feels sorta weird? 

Anyway, if the question you were asking is the first one LMK if you disagree – I can see good arguments both ways, so we could change it for sure.

Reply:
My question was the first one and I think a comment above the exit statement would be enough. I don't have a strong preference either way on that decision.